README.md

```diff
@@ -1 +1,33 @@
-# node-hello-world-lab
\ No newline at end of file
+# Node Hello World
+
+## Objectives
+
+1. Implement Hello World in Node
+1. Run the script from the terminal
+
+
+## Introduction
+
+In this lab, you'll practice how to run Node scripts from command line. In addition, you'll develop your first Node application—Hello World, because such app is a must when learning a new language.
+
+## Instructions
+
+1. Create file `hello-world.js` and open it in your favorite editor
+2. Output "Hello World" with `console.log()` which works similarly to browser JavaScript, i.e., prints text.
+3. Launch the script from command line / terminal / command prompt with `$ node hello-world` (`.js` is optional).
```

Comment:
get rid of the `$` before the `npm test`. we typically only give students what they need to enter in terminal without the prompt. students may take this literally and enter `$ npm test` in terminal